### PR TITLE
(docs) Update Tooltip: Bake Configuration -> Base AMI

### PR DESCRIPTION
**Pull request by ankitml**
As a one day old spinnaker user, I felt the old tooltip was not conveying what it meant properly and this update will help other new comers as well.

### CODE REVIEW

File: app/scripts/modules/core/src/help/help.contents.ts [hunk 1]
```diff
@@ -113,7 +113,7 @@ module(HELP_CONTENTS, [])
     'pipeline.config.docker.bake.targetImage': '<p>The name of the resulting docker image.</p>',
     'pipeline.config.docker.bake.targetImageTag': '<p>The tag of the resulting docker image, defaults to commit hash if available.</p>',
     'pipeline.config.docker.bake.organization': '<p>The name of the organization or repo to use for the resulting docker image.</p>',
-    'pipeline.config.bake.baseAmi': '<p>(Optional) ami-????????</p>',
+    'pipeline.config.bake.baseAmi': '<p>(Optional) If Base AMI is specified, this will be used instead of the Base OS provided',
```
Comment (anotherchrisberry) [inline app/scripts/modules/core/src/help/help.contents.ts]: Should this say "Base image" instead of "Base OS"?

Comment (ankitml) [reply app/scripts/modules/core/src/help/help.contents.ts]: Third field for 'Bake Configuration' form says 'Base OS'. So if we specify this field 'base AMI', bake step ignores what we have provided in 'Base OS field'. That has been my understanding till now and conveying this in the tooltip

Comment (anotherchrisberry) [reply app/scripts/modules/core/src/help/help.contents.ts]: Ah, right, makes sense.

**Comment by anotherchrisberry** [comment]
LGTM

**Pull request merged**